drake-torch.dockerfile

```diff
@@ -187,8 +230,13 @@ RUN cd $HOME && git clone https://github.com/lcm-proj/lcm.git \
 # necessary to make all installed libraries available for linking
 RUN ldconfig
 
-# setup entrypoint
-COPY scripts/docker_entrypoint.sh /root
+# Set debconf back to normal.
+RUN echo 'debconf debconf/frontend select Dialog' | debconf-set-selections
+
+# setup entrypoint (not needed anymore)
+# ADD scripts/docker_entrypoint.sh $HOME/docker_entrypoint.sh
+# RUN chmod +x $HOME/docker_entrypoint.sh
+# CMD ["$HOME/docker_entrypoint.sh"]
 
-ENTRYPOINT ["docker_entrypoint.sh"]
-CMD ["bash"]
+# start ssh daemon
+CMD ["/usr/sbin/sshd", "-D"]
```

Comment:
Why are we running `sshd` inside the docker? Can't we just use `docker exec` to get inside the docker?

Reply:
we like to start it in detached mode. you can just use docker exec to get inside the docker.